toolkit/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/quay/claircore/toolkit
+
+go 1.16
```

Comment:
Bump the go version?

Also, I thought with 1.17 lazy loading of packages was standard, is it required to have this be it'd own module?